h2o-docs/src/product/data-science/algo-params/metalearner_transform.rst

```diff
@@ -0,0 +1,139 @@
+``metalearner_transform``
+-------------------------
+
+- Available in: Stacked Ensembles
+- Hyperparameter: no
```

Comment:
I'm not really sure if it should be a hyper param or not. Metalearner algorithm is not so I used "no" as well.